doc/subsystems/chaos_pp/chaos_pp.rst

```diff
@@ -0,0 +1,916 @@
+..
+    Copyright (c) 2018 Bobby Noelte
+    SPDX-License-Identifier: Apache-2.0
+
+.. _chaos_pp:
+
+Chaos Preprocessor Library
```

Comment:
Oh my.  Not sure we want to document a component in ``/ext`` like this. Isn't there an external repo/site we can refer to rather than, what looks like, duplicating the library documentation here?

Reply:
Sorry, there is no external documentation site.